docs/insights-archive-sample/insights-operator/gathers.json

```diff
@@ -0,0 +1,214 @@
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherClusterImageRegistry",
+            "duration_in_ms": 650,
+            "records_count": 1,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherCRD",
+            "duration_in_ms": 774,
+            "records_count": 2,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherCertificateSigningRequests",
+            "duration_in_ms": 791,
+            "records_count": 0,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherNodes",
+            "duration_in_ms": 920,
+            "records_count": 6,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherOpenShiftAPIServerOperatorLogs",
+            "duration_in_ms": 940,
+            "records_count": 0,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherConfigMaps",
+            "duration_in_ms": 1018,
+            "records_count": 11,
+            "errors": [
+                "configmaps \"cluster-monitoring-config\" not found"
+            ]
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherContainerImages",
+            "duration_in_ms": 1105,
+            "records_count": 10,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherOpenshiftAuthenticationLogs",
+            "duration_in_ms": 1177,
+            "records_count": 0,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherOpenshiftSDNControllerLogs",
+            "duration_in_ms": 1328,
+            "records_count": 0,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherOpenshiftSDNLogs",
+            "duration_in_ms": 2611,
+            "records_count": 0,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherClusterOperators",
+            "duration_in_ms": 3860,
+            "records_count": 49,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherInstallPlans",
+            "duration_in_ms": 10445,
+            "records_count": 1,
+            "errors": null
+        },
+        {
+            "name": "github.com/openshift/insights-operator/pkg/gather/clusterconfig.GatherServiceAccounts",
```

Comment:
I know this is totally unrelated to this PR but I really think the name is just  unnecessarily long (It proved again when working with the data in Jupyter notebooks). I find the prefix `github.com/openshift/insights-operator` useless.